src/main/java/minicraft/screen/OptionsMainMenuDisplay.java

```diff
@@ -20,6 +24,14 @@ public OptionsMainMenuDisplay() {
         );
     }
 
+    @Override
+	public void render(Screen screen) {
+		super.render(screen);
+        
+		// Warning users about screen resoluton requiring a restart

```

Comment:
Would be nice to make this work without restarting the game.

Reply:
It would be, but it would require messing with a LOT of init code.

It would at minimum require another call to
```java
Initializer.createAndDisplayFrame();
// and
Renderer.initScreen();
```

But this would just create another display without closing the old one. It may be worth looking into in the future, but unless it's a dealbreaker for you, I don't think it's worth adding complexity for this.

Reply:
It's not a dealbreaker. I just wanted to make the point.